features/metadata_checks_are_reported_in_a_prometheus_compatible_metrics_endpoint.feature

```diff
@@ -20,56 +20,56 @@ Feature: metadata checks are reported in a prometheus compatible metrics endpoin
     And the metadata is signed by a certificate belonging to SIGNING_PKI_ONE
     And there is metadata at http://localhost:53110
     And there is an OCSP responder
-    When I start the prometheus client on port 2020 with metadata on port 53110 with ca test_pki_one.crt
-    Then the metrics on port 2020 should contain exactly:
+    When I start the prometheus client on port 2040 with metadata on port 53110 with ca test_pki_one.crt
+    Then the metrics on port 2040 should contain exactly:
     """
-    # TYPE verify_metadata_certificate_expiry gauge
-    # HELP verify_metadata_certificate_expiry The NotAfter date of the given X.509 SAML certificate
-    verify_metadata_certificate_expiry{entity_id="foo",use="encryption",key_name="foo_key_1",serial="2",subject="/DC=org/DC=TEST/CN=GENERATED TEST CERTIFICATE"} (\d+).0
-    verify_metadata_certificate_expiry{entity_id="foo",use="encryption",key_name="foo_key_2",serial="3",subject="/DC=org/DC=TEST/CN=GENERATED TEST CERTIFICATE"} (\d+).0
-    verify_metadata_certificate_expiry{entity_id="bar",use="encryption",key_name="bar_key_1",serial="4",subject="/DC=org/DC=TEST/CN=GENERATED TEST CERTIFICATE"} (\d+).0
-    verify_metadata_certificate_expiry{entity_id="metadata_signature",use="",key_name="certificate",serial="2",subject="/DC=org/DC=TEST/CN=SIGNED TEST CERTIFICATE"} (\d+).0
-    # TYPE verify_metadata_expiry gauge
-    # HELP verify_metadata_expiry The validUntil date of the given metadata
-    verify_metadata_expiry{metadata="http://localhost:53110"} 1434720100000.0
-    # TYPE verify_metadata_certificate_ocsp_success gauge
-    # HELP verify_metadata_certificate_ocsp_success If a cert chain validation and OCSP check of the given X.509 SAML certificate is good \(1\) or bad \(0\)
-    verify_metadata_certificate_ocsp_success{entity_id="foo",use="encryption",key_name="foo_key_1",serial="2",subject="/DC=org/DC=TEST/CN=GENERATED TEST CERTIFICATE"} 1.0
-    verify_metadata_certificate_ocsp_success{entity_id="foo",use="encryption",key_name="foo_key_2",serial="3",subject="/DC=org/DC=TEST/CN=GENERATED TEST CERTIFICATE"} 1.0
-    verify_metadata_certificate_ocsp_success{entity_id="bar",use="encryption",key_name="bar_key_1",serial="4",subject="/DC=org/DC=TEST/CN=GENERATED TEST CERTIFICATE"} 1.0
-    verify_metadata_certificate_ocsp_success{entity_id="metadata_signature",use="",key_name="certificate",serial="2",subject="/DC=org/DC=TEST/CN=SIGNED TEST CERTIFICATE"} 1.0
+verify_metadata_certificate_expiry{entity_id="foo",use="encryption",serial="2",subject="/DC=org/DC=TEST/CN=GENERATED TEST CERTIFICATE"} (\d+).0
```

Comment:
These should be indented to the same level as the surrounding `"""` ([source](https://cucumber.io/docs/gherkin/reference/) which says "The indentation inside the triple quotes, however, is significant. Each line of the Doc String will be dedented according to the opening `"""`.")